src/zjs_dgram.c

```diff
@@ -217,22 +215,19 @@ static void udp_sent(struct net_context *context, int status, void *token,
     DBG_PRINT("udp_sent: %p, st=%d udata=%p\n", context, status, user_data);
 
     if (user_data) {
-        jerry_value_t args[1] = {ZJS_UNDEFINED};
+        ZVAL rval = ZJS_UNDEFINED;
         if (status != 0) {
             char errbuf[8];
             snprintf(errbuf, sizeof(errbuf), "%d", status);
-            args[0] = zjs_standard_error(NetworkError, errbuf);
+            rval = zjs_standard_error(NetworkError, errbuf);
             // We need error object, not error value (JrS doesn't allow to
             // pass the latter as a func argument).
-            jerry_value_clear_error_flag(&args[0]);
+            jerry_value_clear_error_flag(&rval);
         }
 
         zjs_callback_id id = zjs_add_callback_once((jerry_value_t)user_data,
-                                                   ZJS_UNDEFINED,
-                                                   NULL,
-                                                   NULL);
-        zjs_signal_callback(id, args, sizeof(args));
-        jerry_release_value(args[0]);
+                                                   ZJS_UNDEFINED, NULL, NULL);
+        zjs_signal_callback(id, &rval, 1);
```

Comment:
Sorry for screwing up your code, @pfalcon. Here's where I went astray... in this massive patch I had a lot of room to screw something up, and I did. I had people review their areas but you weren't around that week. :/

So first I made *this* bug. Then later when I came back to the dgram.c file I think I saw this line and thought "oh, we're passing the number of jerry_values, not bytes", and thought the other one was wrong. So I screwed up a second one!

I'll get a PR in shortly to fix these.